src/main/java/graphql/testrunner/service/GitService.java

```diff
@@ -0,0 +1,50 @@
+package graphql.testrunner.service;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import graphql.testrunner.util.TestRunnerException;
+
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class GitService {
+
+    private static final Logger LOGGER = Logger.getLogger(GitService.class.getName());
+    private static final String REPO_URL = "https://github.com/graphql-java/graphql-java.git";
+
+    /**
+     * Pulls the code from graphql-java library and checkouts the given commit hash.
```

Comment:
repeating a previous ultra nit: this comment doesn't add anything because your code is already quite clear.